Dockerfile

```diff
@@ -34,10 +35,21 @@ ENV OUTPUT_CLOUDWATCH_ENABLED   false
 ENV OUTPUT_KAFKA_ENABLED        false
 
 COPY telegraf.conf.tpl /etc/telegraf/telegraf.conf.tpl
-COPY run.sh /run.sh
```

Comment:
can we keep the run.sh script, and call it (exec) in the start.sh script? Like what is done in the influxdb image